src/app/profile/page.tsx

```diff
@@ -51,18 +53,17 @@ export default function Page() {
         {profiles.map(profile => (
           <li key={profile.user_id}>
             <h2>Profile ID: {profile.user_id}</h2>
-            <p>First Name: {profile.first_name}</p>
-            <p>Last Name: {profile.last_name}</p>
             <p>Roles: {profile.roles.join(', ')}</p>
             <p>Languages: {profile.languages.join(', ')}</p>
```

Comment:
I'm getting an error here on the profiles page: "TypeError: Cannot read properties of null (reading 'join')"